### ISSUE TITLE
MCP6001T in circuit

### ISSUE THREAD

**Issue by maxholgasson** [TeraHz/I2C-LED-Driver#1]
HI, as I didnt find a notice about the MCP6001T.
What's the purpose of this Op Amp and why do you need it?

**Comment by TeraHz** [comment]
The opamp is used in a comparator configuration to turn on/off the LED driver via its UVLO pin. It's been a while, so I can't 100% remember why, but I think it had to do with the I2C DAC (MCP4725) not being able to turn off the LM3408HV or it misbehaving at the lowest voltages.

**Comment by maxholgasson** [comment]
Hi ah ok I see, thanks for your answer! I'm looking for a way to built a no flickering light for filming. Would you recommend that circuit for that purpose? I had some issues with other pwm solutions and could imagine your approach could work? I had a similar circuit in mind with the dac7571 (but don't know if it's a better one)

**Comment by TeraHz** [comment]
Do you need the I2C part of it? If not, I recommend just doing a straight LV3408HV circuit. It's a great driver, I've used it on a few different projects. Here is a cleaned-up circuit for PWM control: https://www.altium.com/viewer?token=zWmvux0NVEqwfgDqUd06ok4l

**Comment by maxholgasson** [comment]
Yes I2C(digital control) and no-flickering are the two key elements
